config/connect-mirror-maker.properties

```diff
@@ -18,18 +18,40 @@
 # Run with ./bin/connect-mirror-maker.sh connect-mirror-maker.properties 
 
 # specify any number of cluster aliases
-clusters = A, B, C
+clusters = A, B 
 
 # connection information for each cluster
-A.bootstrap.servers = A_host1:9092, A_host2:9092, A_host3:9092
-B.bootstrap.servers = B_host1:9092, B_host2:9092, B_host3:9092
-C.bootstrap.servers = C_host1:9092, C_host2:9092, C_host3:9092
+# This is a comma separated host:port pairs for each cluster
+# for e.g. "A_host:9092, A_host2:9092, A_host3:9092"
+A.bootstrap.servers = localhost:9092
+B.bootstrap.servers = localhost:10092
```

Comment:
What about keeping the host/port as they were instead of using locahost for both?

Reply:
This seems better . 
Will revert it.